js/runner.go

```diff
@@ -297,7 +296,16 @@ func (r *Runner) runPart(ctx context.Context, out chan<- stats.SampleContainer,
 	}
 
 	v, _, err := vu.runFn(ctx, group, fn, vu.Runtime.ToValue(arg))
-	cancel()
```

Comment:
I am not sure this `cancel()` is without a reason. Are you sure that there isn't anything that might be left dangling if we remove it? After all, maybe the script execution will finish normally, without cancelling the context, but something will still be left listening to it? If you need access to the original `ctx` below, just rename it?